test/cartesian/Area.spec.tsx

```diff
@@ -440,8 +440,8 @@ describe.each(chartsThatSupportArea)('<Area /> as a child of $testName', ({ Char
 
       const { rerender } = render(
         <ChartElement data={data}>
-          <Area dataKey="value" data={data2} xAxisId={7} yAxisId={9} stackId="q" hide />
-          <Customized component={<Comp />} />
+          <Area dataKey="value" data={data2} xAxisId={7} yAxisId={9} stackId="q" hide id="my-custom-area-id" />
```

Comment:
slight concern that `id` will be conflated with DOM element `id` in props even though this doesn't get passed down to the DOM element.

Maybe a different name would make sense for the public interface? `chartId?`